docs/provisioner/provisioning-api.md

```diff
@@ -87,6 +87,46 @@ The mutation returns OperationID allowing to retrieve the operation status.
 
```

Comment:
![image](https://user-images.githubusercontent.com/53601578/87153713-30fed700-c2b8-11ea-823e-380d48977a9b.png)

I got that as input for the documentation related to this PR and it's already [here](https://github.com/kyma-project/control-plane/pull/40), so the changes in the `provisioning-api.md` file can be removed from this PR.